database/createDatabases.ps1

```diff
@@ -117,11 +124,33 @@ if(!$installed){
 Set-ItemProperty -Path 'HKLM:\\SOFTWARE\\Wow6432Node\\Microsoft\\.NetFramework\\v4.0.30319' -Name 'SchUseStrongCrypto' -Value '1' -Type DWord -Force
 Set-ItemProperty -Path 'HKLM:\\SOFTWARE\\Microsoft\\.NetFramework\\v4.0.30319' -Name 'SchUseStrongCrypto' -Value '1' -Type DWord -Force
 
+#formulate logins for single Database
+$helper_login_user=$dbuser+"_helper"
+$pkgmanager_login_user=$dbuser+"_pkgmanager"
+$deployer_login_user=$dbuser+"_deployer"
+$trainer_login_user=$dbuser+"_trainer"
+$appmanager_login_user=$dbuser+"_appmanager"
+
+
+if($singleDatabase -eq "N"){
+    $sqlcommand = "
+    GO
+    CREATE LOGIN $dbuser WITH PASSWORD=N'{{pwd}}'
+    GO
+    "
+}
+else {
 $sqlcommand = "
-GO
-CREATE LOGIN $dbuser WITH PASSWORD=N'{{pwd}}'
-GO
-"
+    GO
```

Comment:
This would make default value as "Y" which seems wrong to me

Reply:
Nope, if the argument is not passed, it's getting set to "N" at the start of the script 

if($singleDatabase.Length -eq 0){
    write-host "creating multiple databases"
    $singleDatabase = "N"
}